docs/source/releases.rst

```diff
@@ -1,6 +1,25 @@
 Release History
 ###############
 
+v1.?.? (2021-??-??)
```

Comment:
Are you interested in just setting this to v1.11.0 and tagging early? I don't remember any other immediate-priority updates planned in this repo.

```suggestion
v1.11.0 (2021-06-04)
```

Reply:
I don't mind, either way 👍

Reply:
In that case I suggest we do it. Version numbers are cheap and it saves us some work later.